services/admin/src/db_client.ts

```diff
@@ -0,0 +1,273 @@
+import { assert } from '@votingworks/utils';
```

Comment:
Seems like a good candidate to extract at some future time.

Reply:
Definitely, I'll file a tech improvement ticket so we don't forget.

Reply:
#1888